src/main/java/org/terasology/logic/health/HealthCommands.java

```diff
@@ -72,6 +74,73 @@ public String damageWithType(@Sender EntityRef client, @CommandParam("damageType
             return "Specified damage type does not exist.";
         }
     }
+    public void setAll(DamageResistComponent damageResist,int v){
+        damageResist.damageTypes=new HashMap<>(6);
+        damageResist.damageTypes.put("directDamage",v);
+        damageResist.damageTypes.put("drowningDamage",v);
+        damageResist.damageTypes.put("explosiveDamage",v);
+        damageResist.damageTypes.put("healingDamage",v);
+        damageResist.damageTypes.put("physicalDamage",v);
+        damageResist.damageTypes.put("supportRemovedDamage",v);
+    }
+    @Command(shortDescription = "Damage Resistance", runOnServer = true,
+            requiredPermission = PermissionManager.CHEAT_PERMISSION)
+    public String damageResist(@CommandParam("DamageType") String dType,@CommandParam("Percentage") int value,@Sender EntityRef clientEntity) {
+        if(value>=0&&value<=100) {
+            ClientComponent player = clientEntity.getComponent(ClientComponent.class);
+            DamageResistComponent damageResist = player.character.getComponent(DamageResistComponent.class);
+            String s="ALL";
```

Comment:
Another example of missing spaces here and elsewhere, also try to avoid single character variable names - make them express more meaning :-)